presto-docs/src/main/sphinx/release/release-0.237.rst

```diff
@@ -0,0 +1,62 @@
+=============
+Release 0.237
+=============
+
+General Changes
+_______________
+* Fix NPE in common sub-expression optimization triggered by CASE-WHEN expression.
+* Fix compiler failure with function type when common sub-expression optimization is enabled.
+* Add JSON format for `EXPLAIN` with type `LOGICAL` and `DISTRIBUTED`.
+* Add functions :func:`array_sum` and :func:`map_normalize`.
+* Add new session property ``warning_handling `` to control how warnings are handled. The options are ``SUPPRESS``, ``NORMAL`` and ``AS_ERROR``. The default value is ``NORMAL``.
+* Add support for Druid's basic and kerberos authentication.
+* Add support for defining SQL-invoked functions in plugins.
+* Added a new aggregation function SET_AGG<T> that returns ARRAY<T> - equivalent to ARRATY_AGG(DISTINCT).
+* COUNT(DISTINCT) suggests using approx_distinct instead.
+* Introducing NodeStatusService interface to find out if a worker is in maintenance mode or ready to take tasks. Default behavior is no op. But can be implemented and override the
+  behavior. Config controlling the behavior `node-status-service.enabled`.
+* Optional NodeStatusService to check Nodes status (i.e. Available/In Maintenance) through some external source and mark them Inactive if not available to process queries.
+* Support listing functions whose names match a specified pattern using the SHOW FUNCTION LIKE syntax.
+* TaskStatus JSON decoding improved about 15% CPU by making it leaner.
+
+Cassandra Changes
+_________________
+* Fix missing Netty library introduced in version 0.229.
+
+Elasticsearch Changes
+_____________________
+* Fix predicate pushdown for Elasticsearch.
+* Add Elasticsearch array support using definitions in the _meta field.
+* Add support for AWS IAM authorization to Elasticsearch connector.
+* Add support for Elasticsearch query string syntax.
+* Add support for nested types in Elasticsearch.
+* Add support for querying Elasticsearch aliases.
+* Add system.nodes table to Elasticsearch.
+* Handle mixed-case columns in Elasticsearch.
+* Load tables dynamically in Elasticsearch.
+* Refresh Elasticsearch nodes periodically.
+* Update Elasticsearch connector to use Elasticsearch Http client.
+* Use shard primary host in Elasticsearch connector to save extra hop in Elasticsearch.
+
+Geospatial Changes
+__________________
+* Add `flatten_geometry_collections` function to recursively flatten `GeometryCollection`\s.
```

Comment:
extra `\s` at the end.

Reply:
I think the note is referring to GeometryCollection objects in plural, like GeometryCollections. The \ is added to prevent sphinx from raising an error.